### PR TITLE
Reduce startup states.

### LINKED ISSUE #137: Reduce startup states
This slowed-down clip of switching to Notes shows about six different states. Is there a way that we can reduce this to one?
[slowmo.mp4.zip](https://github.com/mozilla/notes/files/1143208/slowmo.mp4.zip)

### PR DESCRIPTION
**Pull request by Natim**
Wait for a full loading before showing the editor. Fixes #137

### CODE REVIEW

File: src/sidebar/styles.css [hunk 1]
```diff
@@ -15,6 +15,16 @@ body {
   padding: 0;
 }
 
+div#loading {
+	height: 200%;
```
Comment (Natim) [inline src/sidebar/styles.css]: can probably use 100% here.

File: src/sidebar/panel.js [hunk 2]
```diff
@@ -121,13 +121,19 @@ function handleLocalContent(data) {
 }
 
 function loadContent() {
-  browser.storage.local.get(['notes'], data => {
-    // If we have a bearer, we try to save the content.
-    handleLocalContent(data);
+  return new Promise((resolve) => {
+    browser.storage.local.get(['notes'], data => {
```
Comment (vladikoff) [inline src/sidebar/panel.js]: Can this fail? Can we log to GA that it failed?

File: src/sidebar/styles.css [hunk 3]
```diff
@@ -15,6 +15,16 @@ body {
   padding: 0;
 }
 
+div#loading {
+	height: 100%;
```
Comment (vladikoff) [inline src/sidebar/styles.css]: indent seems wrong here

File: src/sidebar/panel.js (hunk 2 above)
Comment (Natim) [inline src/sidebar/panel.js]: No it cannot fail.

**Pull request merged**